README.md

```diff
@@ -7,14 +7,21 @@ For pre-commit: see https://github.com/pre-commit/pre-commit
 
 For clang-format: see https://github.com/ssciwr/clang-format-wheel
 
+In comparison to other available pre-commit hooks for clang-format, this one installs clang-format from Python wheels.
+This makes it usable without additional installation requirements - e.g. in pre-commit CI.
```

Comment:
I think the `clang-format-wheel` README goes over this enough in details that we don't need to dupe those docs here